_component_design/choice-tiles.md

```diff
@@ -161,18 +170,22 @@ Use the following column widths for the small, medium, and large choice tiles. B
 
 {% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
 
-### Basic states
+#### Single-select interactions
+
+A single-select interaction is analogous to radios where the user can only select one choice tile at a time.
+
+In the default state, the user can click any of the choice tiles. You can construct choices tiles so that one of the options is selected by default. The user can then make another selection, if necessary. When the user selects an option, the previously selected choice tile becomes deselected and all remaining tiles change to the subdued state. The subdued state reinforces that only one choice tile can be selected.
```

Comment:
Also, we should definitely add a line about the possibility of  having one tile selected by default. you can look to radios for the verbiage we used there.